index/miner/types.go

```diff
@@ -6,20 +6,21 @@ import (
 
 // IndexSnapshot contains on-chain and off-chain information about miners
 type IndexSnapshot struct {
-	Meta  MetaIndex
-	Chain ChainIndex
+	Meta    MetaIndex
+	OnChain ChainIndex
 }
 
 // ChainIndex contains on-chain information about miners
 type ChainIndex struct {
 	LastUpdated int64
-	Power       map[string]Power
+	Miners      map[string]OnChainData
 }
```

Comment:
So here is a change in structure of the index data. 
Instead of having data _per-dimension_ ("Power", "SectorSize", "ActiveDeals"), sounds much more useful to group the data by miner. Seems more natural. Unfortunately, that change scattered to many files but sounds like worth it.

Reply:
Yea good call, much better for consuming this way. Give you the total miner view in a simple way then it's easy to map to lists of single types.